stix_shifter_modules/azure_sentinel/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -443,15 +443,15 @@
         "key": "process.pid"
       },
       "valueData": {
-        "key": "extensions.windows-registry-value-type.valueData",
+        "key": "windows-registry-value-type.data",
```

Comment:
Would these need to actually be `windows-registry-key.windows-registry-value-type`?

Reply:
i think it should be `windows-registry-key.values.data`. This is what i understand from the doc and this example-
```
{
  "type": "windows-registry-key",
  "spec_version": "2.1",
  "id": "windows-registry-key--2ba37ae7-2745-5082-9dfd-9486dad41016",
  "key": "hkey_local_machine\\system\\bar\\foo",
  "values": [
    {
      "name": "Foo",
      "data": "qwerty",
      "data_type": "REG_SZ"
    },
    {
      "name": "Bar",
      "data": "42",
      "data_type": "REG_DWORD"
    }
  ]
}

```

Reply:
The proper STIX paths would be e.g. `windows-registry-key:values[*].name` since `values` is a list, but the mappings don't really follow that notation.  Do you need the "group" key in the mapping?

Reply:
I'm making use of the "group" value's name in the mapping to support this, it's handled a little differently from the group boolean value currently used.